src/main/java/com/ericsson/ei/frontend/utils/BackEndInformationControllerUtils.java

```diff
@@ -52,7 +50,7 @@
      */
     public ResponseEntity<String> handleRequestForInstances(HttpServletRequest request) {
```

Comment:
Please rename method to getBackEndInstances or something.

Reply:
As it is implemented right now, getBackendInstances would be a lie as it actually creates a response based upon the retrieved backend instances. I don't want to create bigger refactoring at the moment but i think it should for example return backendinstances and then the controller should create the response object.

Reply:
I will add a refactoring story about this together with whats mentioned above.